warehouse/query-core/src/main/java/datawave/query/jexl/visitors/PushdownLargeFieldedListsVisitor.java

```diff
@@ -187,6 +238,26 @@ public Object visit(ASTOrNode node, Object data) {
         return children(newNode, children.toArray(new JexlNode[children.size()]));
     }
     
+    /**
+     * If data is set update it for the given field to include the new possible reduction, otherwise do nothing
+     * 
+     * @param data
+     * @param field
+     * @param reduction
+     */
+    private void track(Object data, String field, int reduction) {
+        if (data instanceof Map) {
+            Map trackingMap = (Map) data;
+            Integer count = 0;
+            if (trackingMap.get(field) != null) {
+                count = (Integer) trackingMap.get(field);
+            }
```

Comment:
IDE warnings are annoying...

```suggestion
            @SuppressWarnings("unchecked")
            Map<String,Integer> trackingMap = (Map<String,Integer>) data;
            Integer count = 0;
            if (trackingMap.get(field) != null) {
                count = trackingMap.get(field);
            }
```

Reply:
Could you add this in please?

Reply:
ok